kura/maven-central/p2tom2.properties

```diff
@@ -0,0 +1,76 @@
+# mirror.<iu>=false
+# dependency.<ns>.<name>.ignore = true|false
+# mapping.<iu>=groupId:artifactId[:version]
+# external.<iu>=groupId:artifactId[:version]
+
+# Select default mapping strategy
+
+implementation.mavenMapping=org.eclipse.scada.releng.p2.to.maven.DefaultMavenMapping
+scrubJars=true
+
+# Define project metadata
+
+default.project.url=http://eclipse.org/kura
+
+developers=ctron
+
+developer.ctron.name=Jens Reimann
+developer.ctron.email=jreimann@redhat.com
+developer.ctron.url=https://dentrassi.de
+developer.ctron.organization=Red Hat Inc
+developer.ctron.organizationUrl=https://redhat.com
+
```

Comment:
I would like to see Eurotech added as a developer organization. I believe the other fields could be left blank if we don't won't to name specific committers. The 'name' attribute could be set to "Various" with the other fields empty.